integration-tests/jpa-mariadb/src/test/java/io/quarkus/it/jpa/mariadb/OfflineTest.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.it.jpa.mariadb;
+
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+public class OfflineTest {
```

Comment:
I see only mariadb's dialect settings are tested, is that on purpose (cost/benefit tradeoff) or did you forget?

Reply:
See https://github.com/quarkusio/quarkus/pull/48130#issuecomment-3144410622